kura/distrib/src/main/resources/common/customize_kura_properties.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2024 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import sys
+import logging
+import os
+from network_tools import get_eth_wlan_interfaces_names
+
+def main():
+    logging.basicConfig(
+        format='[customize_kura_properties.py] %(asctime)s %(levelname)s %(message)s',
+        level=logging.INFO,
+        datefmt='%Y-%m-%d %H:%M:%S',
+        handlers=[
+            logging.StreamHandler()
+        ]
+    )
+    
+    board_name = sys.argv[1:] # remove script name from args
```

Comment:
This is somewhat misleading. `board_name` here is not a string as the name might suggest but a slice of the original `sys.argv` array.

I would either change its name to `cli_args` (or something similar) or directly extract the board_name (risky):

```suggestion
    board_name = sys.argv[1:][0] # remove script name from args
```